Makefile

```diff
@@ -9,10 +10,20 @@ fmt: ${HOME}/.cargo/bin/cargo-fmt
 build:
 	RUSTFLAGS='-D warnings' cargo build
 
+# Tests are in order of complexity, from least to greatest.
+test-loop:
+	sudo env "PATH=${PATH}" RUSTFLAGS='-D warnings' cargo test -- --test test_force_flag_stratis
+	sudo env "PATH=${PATH}" RUSTFLAGS='-D warnings' cargo test -- --test test_linear_device
+	sudo env "PATH=${PATH}" RUSTFLAGS='-D warnings' cargo test -- --test test_thinpool_device
+	sudo env "PATH=${PATH}" RUSTFLAGS='-D warnings' cargo test -- --test test_pool_blockdevs
```

Comment:
can you put in the commitlog how the four old tests match up with the newly named four tests? Also, just generally discuss how the new test implementations differ from the old tests. They should differ in style but still cover exactly the same cases, is this true?